.github/workflows/checks.yml

```diff
@@ -18,8 +18,8 @@ jobs:
       - uses: actions/checkout@v4
       - uses: subosito/flutter-action@v2.16.0
         with:
+          flutter-version: '3.13.0' # Uses the minimum supported version.
```

Comment:
What is the reason we are switching back to this way?

Reply:
Please see https://github.com/woltapp/wolt_modal_sheet/pull/219#discussion_r1623630183

Reply:
We cannot use automatic pubspec version if we use constraints.